frontend/integration-tests/tests/kubevirt/models/virtualMachine.ts

```diff
@@ -25,7 +25,10 @@ export class VirtualMachine extends DetailView {
   }
 
   async action(action: string, waitForAction?: boolean) {
-    await this.navigateToTab(vmView.overviewTab);
+    const currentUrl = await browser.getCurrentUrl();
```

Comment:
The intention here is making it possible to call vm.action on project status vm action page. I agree that it might break something. I will change the condition to `Don't navigate to overviewTab` when it's on project status page.

Reply:
Right, I agree. If I may suggest, I think it would be a good solution to check that `$('.co-m-pane__name--overview')` contains `Project Status` and that `$('.overview__sidebar .co-resource-item__resource-name').getText()` equals `this.name`, which would make sure that the right vm is selected in the project status page.
If this condition is not met, then we can navigate to overviewTab.
I suppose that in the long term I should implement a class representing the Project Status, that would have methods to work with the resources available there.

Reply:
I revert the changes by calling `detailViewAction` directly.

The ideal situation is moving `await this.navigateToTab(vmView.overviewTab)` out of `action` and `waitForStatusIcon`, but it need to add the line previous to every caller.

I'm okay to use `detailViewAction`.